lib/transloadit.rb

```diff
@@ -8,6 +8,7 @@
 class Transloadit
   autoload :ApiModel, 'transloadit/api_model'
   autoload :Assembly, 'transloadit/assembly'
+  autoload :Exception, 'transloadit/exceptions'
```

Comment:
I think the module name should be resembled in the filename, so either we rename the file to `exception`, or we rename the constant to `Exceptions` :)